docs/dev-running.md

```diff
@@ -1,25 +1,8 @@
-## Running Fedimint for dev testing
-
-Fedimint consists of three kinds of executables:
-
-* **Federation nodes** - servers who form the mint by running a consensus protocol
-* **Lightning gateways** - allows users send and receive over Lightning by bridging between the mint and an LN node
-* **User clients** - handles user communication with the mint and the gateway
-
 ### Prerequisites
 
-In order to run Fedimint you will need:
-- The [Rust toolchain](https://www.rust-lang.org/tools/install) to build and run the executables
-- The [Nix package manager](https://nixos.org/download.html) for managing build and test dependencies
-
-Clone and `cd` into the Fedimint repo:
-
-```shell
-git clone git@github.com:fedimint/fedimint.git
-cd fedimint
-```
+Fedimint's developer environment is build around the [Nix package manager](https://nixos.org). Once setup, you can execute one `nix develop` command to create a completely deterministic development, test and build environment with all required dependencies installed and pinned to exact versions. Detailed setup instructions are available in [dev-env.md](./dev-env.md).
```

Comment:
Old instructions kinda suggested that they should install rust outside nix. Let's be more clear about forcing them to install nix, but also explain why it's worth it.